src/utils/assertionutil.js

```diff
@@ -0,0 +1,168 @@
+export function IsWebAuthnSupported() {
```

Comment:
should all the functions be exported?

Reply:
Removed unnecessary export and renamed it to fidoFlowUtil.js.